File: web/web service/DAO/Producto.php
<?php  

class Producto {

	//Atributos
	private $idProducto;
    private $nombre;
    private $marca;
    private $categoria;
    private $existencia;
	private $precio;
    private $foto;

	//Constructores
	function __construct() {

	}

	function Producto($nombre, $marca, $categoria, $existencia, $precio, $foto) {
		this::$nombre = $nombre;
		this::$marca = $marca;
		this::$categoria = $categoria;
		this::$existencia = $existencia;
		this::$precio = $precio;
		this::$foto = $foto;
	}

	public static function getIdProducto() {
		return this::$idProducto;
	}

	public static function setIdProducto($idProducto) {
		this::$idProducto = $idProducto;
	}

	public static function getNombre() {
		return this::$nombre;
	}

	public static function setNombre($nombre) {
		this::$nombre = $nombre;
	}

	public static function getMarca() {
		return this::$marca;
	}

	public static function setMarca($marca) {
		this::$marca = $marca;
	}

	public static function getCategoria() {
		return this::$categoria;
	}

	public static function setCategoria($categoria) {
		this::$categoria = $categoria;
	}

	public static function getExistencia() {
		return this::$existencia;
	}

	public static function setExistencia($existencia) {
		this::$existencia = $existencia;
	}

	public static function getPrecio() {
		return this::$precio;
	}

	public static function setPrecio($precio) {
		this::$precio = $precio;
	}

	public static function getFoto() {
		return this::$foto;
	}

	public static function setFoto($foto) {
		this::$foto = $foto;
	}

	//Métodos constructores

}

?>

File: app/DDD-Market.docx/app/src/main/java/com/example/ddd_market/ddd_market/modelo/DAO/Producto.java
package com.example.ddd_market.ddd_market.modelo.DAO;

/**
 *
 * @author DDD
 */

public class Producto {

    //<editor-fold defaultstate="collapsed" desc="Atributos">
    private int idProducto;
    private String nombre;
    private String marca;
    private String categoria;
    private int existencia;
    private double precio;
    private String foto;
    //</editor-fold>

    //<editor-fold defaultstate="collapsed" desc="Constructores">
    /**
     * Constructor por defecto
     */
    public Producto() {
    }

    /**
     * Constructor sobrecargado
     *
     * @param nombre
     * @param marca
     * @param categoria
     * @param existencia
     * @param precio
     * @param foto
     */
    public Producto(String nombre, String marca, String categoria, int existencia, double precio, String foto) {
        this.nombre = nombre;
        this.marca = marca;
        this.categoria = categoria;
        this.existencia = existencia;
        this.precio = precio;
        this.foto = foto;
    }
    //</editor-fold>

    //<editor-fold defaultstate="collapsed" desc="Métodos get y set">
    /**
     * Método get de id de producto
     *
     * @return id de producto
     */
    public int getIdProducto() {
        return idProducto;
    }

    public void setIdProducto(int idProducto) {
        this.idProducto = idProducto;
    }

    /**
     * Método get de nombre
     *
     * @return nombre
     */
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * Método get de marca
     *
     * @return marca
     */
    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    /**
     * Método get de categoria
     *
     * @return categoria
     */
    public String getCategoria() {
        return categoria;
    }

    public void setCategoria(String categoria) {
        this.categoria = categoria;
    }

    /**
     * Método get de existencia
     *
     * @return existencia
     */
    public int getExistencia() {
        return existencia;
    }

    public void setExistencia(int existencia) {
        this.existencia = existencia;
    }

    /**
     * Método get de precio
     *
     * @return precio
     */
    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }

    /**
     * Método get de foto
     *
     * @return foto
     */
    public String getFoto() {
        return foto;
    }

    public void setFoto(String foto) {
        this.foto = foto;
    }
    //</editor-fold>

}
